svt-av1/PKGBUILD

```diff
@@ -19,22 +20,17 @@ makedepends=(
   nasm
   ninja
 )
-_tag=59645eea34e2815b627b8293aa3af254eddd0d69
-source=(git+https://gitlab.com/AOMediaCodec/SVT-AV1.git#tag=${_tag})
+_tag=5cac47c1eaee9c5229cdf693bf26e90ff6c54b40
+source=(git+https://github.com/gianni-rosato/svt-av1-psy.git#tag=${_tag})
 b2sums=('SKIP')
 
 prepare() {
-  sed '/CMAKE_BUILD_TYPE Release/d' -i SVT-AV1/CMakeLists.txt
-}
-
-pkgver() {
-  cd SVT-AV1
-  git describe --tags | sed 's/^v//'
+  sed '/CMAKE_BUILD_TYPE Release/d' -i svt-av1-psy/CMakeLists.txt
 }
 
 build() {
   export LDFLAGS="$LDFLAGS -Wl,-z,noexecstack"
-  cmake -S SVT-AV1 -B build -G Ninja \
+  cmake -S svt-av1-psy -B build -G Ninja \
```

Comment:
Enable AVX512 may be a problem. Either we define two subdirs as in the blas package, or we check for the architecture before building.

Reply:
@lseman no, it's not a problem. ENABLE_AVX512 enables building avx512 variants of functions. 
the dispatch takes care which impl to use if cpu supports it